Source/AviationWeather.Test/Source/metar_parser_tests.cpp

```diff
@@ -352,7 +355,22 @@ void MetarParserTests::METAR_Parser_RunwayVisualRange()
     Assert::AreEqual(uint8_t(27), it->runway_number);
     Assert::AreEqual(uint16_t(6000), it->visibility_min);
     Assert::AreEqual(uint16_t(6000), it->visibility_max);
-    Assert::AreEqual(metar::visibility_modifier_type::greater_than, it->visibility_modifier);
+    Assert::AreEqual(metar::visibility_modifier_type::greater_than, it->visibility_min_modifier);
+    Assert::AreEqual(metar::visibility_modifier_type::none, it->visibility_max_modifier);
+    metar.runway_visual_range_group.clear();
+
+    // R01L/L0600VM1000FT
```

Comment:
Comment doesn't match string.  Also I'm not sure it makes sense to have P1000 since it would seem that values up to 6000 are explicitly quantifiable in many METARs with RVR.  Also, would an actual METAR have a leading 0 on three digit RVR distances?